helm/cloud-provider-azure/templates/cloud-provider-azure.yaml

```diff
@@ -492,7 +493,9 @@ spec:
         image: {{ template "image.cloudNodeManager" . | required "you must use a supported version of Kubernetes or provide cloudNodeManager.imageRepository, cloudNodeManager.imageName, and cloudNodeManager.imageTag values" }}
         imagePullPolicy: {{ .Values.cloudNodeManager.imagePullPolicy }}
         command: ["powershell.exe"]
-        args: ["$env:CONTAINER_SANDBOX_MOUNT_POINT/scripts/start.ps1"]
+        args:
+          - "$env:CONTAINER_SANDBOX_MOUNT_POINT/scripts/start.ps1"
+          - "--v={{ .Values.cloudNodeManager.logVerbosity }}"
```

Comment:
@lzhecheng is this the correct way to configure log verbosity for cloud-node-manager-windows?

cc @marosset @jsturtevant

Reply:
It doesn't seem to be. Since the entrypoint is the start.ps1, I think an parameter for this script is needed and such parameter will be passed to cloud-node-manager.exe as an option.

Reply:
Definitely. Does that parameter not exist in the current Windows implementation of cloud-node-manager for Windows?

Reply:
No, so far there isn't one.